docs/docs/reference/shards/lisp/misc.md

```diff
@@ -33,44 +33,92 @@ This is useful for cases where you need multiple shards (to transform your data)
 
 ## apply
 
-```clojure linenums="1"
+```
+(apply f args) (apply f x args) (apply f x y args) (apply f x y z args)(apply f a b c d & args)
+```
+
+Applies fn f to the argument list formed by prepending intervening arguments to args.
 
+```clojure linenums="1"
+(apply str ["cherry " "banana " "apple"])
```

Comment:
Can you give an example with multiple arguments as well?
I got `(apply str "cherry " "banana " "apple" [])` to work but `(apply str "cherry " "banana " "apple")` doesn't.
I'm still a bit confused what this does.

Reply:
How about now?

Reply:
Yeah looks good